ldms/src/ldmsd/ldmsd.c

```diff
@@ -2215,7 +2215,7 @@ int main(int argc, char *argv[])
 			myhostname[0] = '\0';
 		listen = (ldmsd_listen_t)ldmsd_cfgobj_first(LDMSD_CFGOBJ_LISTEN);
 		snprintf(myname, sizeof(myname), "%s:%d",
-				myhostname, listen->port_no);
+				myhostname, listen ? listen->port_no : -1);
```

Comment:
I've seen this same bug. Should the port_no be zero though?

Reply:
Since 0 is ambiguous but still suggests a listener, my first inclination if -1 seems inappropriate is to add a separate print omitting the ':%d', @tom95858.  Happy to do that if you prefer.